scripts/gen_cfb_font_header.py

```diff
@@ -75,7 +75,7 @@ def generate_header():
         if arg.startswith("--bindir"):
             # Drop. Assumes --bindir= was passed with '=' sign.
             continue
-        if arg.startswith(args.bindir):
+        if (len(args.bindir) > 0) and arg.startswith(args.bindir):
```

Comment:
My bad I created `--bindir` optional but I didn't actually test without passing it.

Can you do this instead? I think it will be clearer. I tested both cases this time.
I can also submit the patch myself if you prefer, up to you.

```diff
--- a/scripts/gen_cfb_font_header.py
+++ b/scripts/gen_cfb_font_header.py
@@ -75,7 +75,7 @@ def generate_header():
         if arg.startswith("--bindir"):
             # Drop. Assumes --bindir= was passed with '=' sign.
             continue
-        if arg.startswith(args.bindir):
+        if args.bindir and arg.startswith(args.bindir):
             # +1 to also strip '/' or '\' separator
             striplen = min(len(args.bindir)+1, len(arg))
             clean_cmd.append(arg[striplen:])
@@ -158,7 +158,7 @@ def parse_args():
         "-o", "--output", type=argparse.FileType('w'), default="-", metavar="FILE",
         help="CFB font header file (default: stdout)")
     group.add_argument(
-        "--bindir", type=str, default="",
+        "--bindir", type=str,
         help="CMAKE_BINARY_DIR for pure logging purposes. No trailing slash.")
     group.add_argument(
         "-x", "--width", required=True, type=int,
```

Reply:
Wow, that really opens up a whole series of potential changes: fix stripping of partial prefixes (`--bindir=/tmp/f -o /tmp/foo/bar` => `-o oo/bar`), fix when `len(os.path.sep)` isn't 1, fix dropping when using `--bindir path`, fix when passed value ends with `os.path.sep` contrary to argument documentation, change name to `--strip-prefix` so it's more general and reflects what it does rather than a name tied to the build system, ....

I pulled back and just made the change you suggested.